deployment/live/aws/test/README.md

```diff
@@ -0,0 +1,222 @@
+# AWS TesseraCT Test Environment
+
+> [!CAUTION]
+> 
+> This test environment creates real Amazon Web Services resources running in your account. They will cost you real money. For the purposes of this demo, it is strongly recommended that you create a new account so that you can easily clean up at the end.
+
+## Prerequisites
+
```

Comment:
This instructions don't mention anything about VPC, which I think is necessary otherwise I don't think the VM can reach the S3 bucket.